app/src/main/java/com/fastlink/zekrapp/di/AppModule.kt

```diff
@@ -1,24 +1,27 @@
-package com.fastlink.zekrapp.appData
+package com.fastlink.zekrapp.di
 
+import android.content.Context
 import dagger.Module
 import dagger.Provides
 import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
 import dagger.hilt.components.SingletonComponent
-import javax.inject.Singleton
 
 @Module
 @InstallIn(SingletonComponent::class)
-object SingletonModule {
+object AppModule {
 
     @Provides
-    @Singleton
     fun provideZekirCategorySingleton(): ZekirCategorySingleton {
```

Comment:
As I mentioned before, objects are already singletons in Kotlin. No need to provide them. Remove both of the functions.